pom.xml

```diff
@@ -1569,10 +815,6 @@
                         <systemPropertyVariables>
                             <org.hibernate.search.test.property-set-in-maven-config>someValue</org.hibernate.search.test.property-set-in-maven-config>
                             <com.sun.management.jmxremote>true</com.sun.management.jmxremote>
```

Comment:
```suggestion
````
Unrelated to your PR, but I think both of these properties can be removed. As far as I can tell they're not useful.